examples/voxceleb/v2/local/extract_vox.sh

```diff
@@ -24,11 +27,13 @@ count=${#data_name_array[@]}
 for i in $(seq 0 $(($count - 1))); do
   bash tools/extract_embedding.sh --exp_dir ${exp_dir} \
     --model_path $model_path \
-    --data_scp ${data_path_array[$i]} \
+    --data_scp ${data_scp_path_array[$i]} \
```

Comment:
Is `data_scp` required for UIO?

Reply:
The data_scp  is used to obtain the number of utterances, so as to judge whether all utterances have been extracted successfully.